### PR TITLE
Refresh virtualization-hpe-simplivity-restapi

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh virtualization-hpe-simplivity-restapi

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 1]
```diff
@@ -84,163 +105,405 @@ Il apporte les Modèles de Service suivants :
...
-    physical drive '1:1-4' status: green
-    physical drive '1:1-5' status: green
-    physical drive '1:1-6' status: green
-    physical drive '3:1-1' status: green
-    physical drive '3:1-2' status: green
-    physical drive '3:1-3' status: green
-    physical drive '3:1-4' status: green
-    physical drive '3:1-5' status: green
-    physical drive '3:1-6' status: green
+OK: alive: 86 faulty: 20 managed: 8 removed: 16 suspected: 31 unknown: 48 green: 3 yellow: 42 red: 12 unknown: 6 logical drives are ok physical drives are ok | 'hosts.alive.count'=86;;;;'hosts.faulty.count'=20;;;;'hosts.managed.count'=8;;;;'hosts.removed.count'=16;;;;'hosts.suspected.count'=31;;;;'hosts.unknown.count'=48;;;;'*hosts*~host.components.green.count'=3;;;;'*hosts*~host.components.yellow.count'=42;;;;'*hosts*~host.components.red.count'=12;;;;'*hosts*~host.components.unknown.count'=6;;;;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:356]: the old one was more complete, is ist normal?

File: pp/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 2]
```diff
@@ -5,70 +5,96 @@ title: HPE Simplivity Rest API
...
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Virt-Hpe-Simplivity-Restapi-custom" label="Virt-Hpe-Simplivity-Restapi-custom">
+
+| Service Alias      | Service Template                                      | Service Description      | Discovery  |
+|:-------------------|:------------------------------------------------------|:-------------------------|:----------:|
+| Hosts              | Virt-Hpe-Simplivity-Hosts-Restapi-custom              | Check hosts              | X          |
+| Omnistack-Clusters | Virt-Hpe-Simplivity-Omnistack-Clusters-Restapi-custom | Check omnistack clusters |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:24]:
```suggestion
| Omnistack-Clusters | Virt-Hpe-Simplivity-Omnistack-Clusters-Restapi-custom | Check Omnistack clusters |            |
```

File: pp/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 3]
```diff
@@ -78,163 +104,405 @@ It brings the following Service Templates:
...
-    --plugin=apps::virtualization::hpe::simplivity::restapi::plugin \
-    --mode=hosts \
-    --hostname='10.0.0.1' \
-    --api-username='my-username' \
-    --api-password='my-password' \
-    --verbose
+	--plugin=apps::virtualization::hpe::simplivity::restapi::plugin \
+	--mode=hosts \
+	--hostname='10.0.0.1' \
+	--api-username='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:312]: username and password

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 4]
```diff
@@ -84,163 +105,405 @@ Il apporte les Modèles de Service suivants :
...
-    --plugin=apps::virtualization::hpe::simplivity::restapi::plugin \
-    --mode=hosts \
-    --hostname='10.0.0.1' \
-    --api-username='my-username' \
-    --api-password='my-password' \
-    --verbose
+	--plugin=apps::virtualization::hpe::simplivity::restapi::plugin \
+	--mode=hosts \
+	--hostname='10.0.0.1' \
+	--api-username='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:313]:
```suggestion
	--api-username='XXXX' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 5]
```diff
@@ -84,163 +105,405 @@ Il apporte les Modèles de Service suivants :
...
+	--api-password='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:314]:
```suggestion
	--api-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 6]
```diff
@@ -78,163 +104,405 @@ It brings the following Service Templates:
...
+	--api-password='' \
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:313]:
```suggestion
	--api-password='XXXX' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md [hunk 7]
```diff
@@ -84,163 +105,405 @@ Il apporte les Modèles de Service suivants :
...
-    physical drive '1:1-4' status: green
-    physical drive '1:1-5' status: green
-    physical drive '1:1-6' status: green
-    physical drive '3:1-1' status: green
-    physical drive '3:1-2' status: green
-    physical drive '3:1-3' status: green
-    physical drive '3:1-4' status: green
-    physical drive '3:1-5' status: green
-    physical drive '3:1-6' status: green
+OK: alive: 86 faulty: 20 managed: 8 removed: 16 suspected: 31 unknown: 48 green: 3 yellow: 42 red: 12 unknown: 6 logical drives are ok physical drives are ok | 'hosts.alive.count'=86;;;;'hosts.faulty.count'=20;;;;'hosts.managed.count'=8;;;;'hosts.removed.count'=16;;;;'hosts.suspected.count'=31;;;;'hosts.unknown.count'=48;;;;'*hosts*~host.components.green.count'=3;;;;'*hosts*~host.components.yellow.count'=42;;;;'*hosts*~host.components.red.count'=12;;;;'*hosts*~host.components.unknown.count'=6;;;;
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md:356]:
```suggestion
OK: All hosts are ok | 'hosts.alive.count'=4;;;; 'hosts.faulty.count'=0;;;; 'hosts.managed.count'=0;;;; 'hosts.removed.count'=0;;;; 'hosts.suspected.count'=0;;;; 'hosts.unknown.count'=0;;;; 'svt01.acme.com#host.components.green.count'=15;;;; 'svt01.acme.com#host.components.yellow.count'=0;;;; 'svt01.acme.com#host.components.red.count'=0;;;; 'svt01.acme.com#host.components.unknown.count'=0;;;; 'svt02.acme.com#host.components.green.count'=15;;;; 'svt02.acme.com#host.components.yellow.count'=0;;;; 'svt02.acme.com#host.components.red.count'=0;;;; 'svt02.acme.com#host.components.unknown.count'=0;;;; 'svt11.acme.com#host.components.green.count'=15;;;; 'svt11.acme.com#host.components.yellow.count'=0;;;; 'svt11.acme.com#host.components.red.count'=0;;;; 'svt11.acme.com#host.components.unknown.count'=0;;;; 'svt12.acme.com#host.components.green.count'=15;;;; 'svt12.acme.com#host.components.yellow.count'=0;;;; 'svt12.acme.com#host.components.red.count'=0;;;; 'svt12.acme.com#host.components.unknown.count'=0;;;;
checking host 'svt01.acme.com'
    status: alive
    logical drive '1' status: green
    logical drive '3' status: green
    physical drive '1:1-1' status: green
    physical drive '1:1-2' status: green
    physical drive '1:1-3' status: green
    physical drive '1:1-4' status: green
    physical drive '1:1-5' status: green
    physical drive '1:1-6' status: green
    physical drive '3:1-1' status: green
    physical drive '3:1-2' status: green
    physical drive '3:1-3' status: green
    physical drive '3:1-4' status: green
    physical drive '3:1-5' status: green
    physical drive '3:1-6' status: green
checking host 'svt02.acme.com'
    status: alive
    logical drive '1' status: green
    logical drive '3' status: green
    physical drive '1:1-1' status: green
    physical drive '1:1-2' status: green
    physical drive '1:1-3' status: green
    physical drive '1:1-4' status: green
    physical drive '1:1-5' status: green
    physical drive '1:1-6' status: green
    physical drive '3:1-1' status: green
    physical drive '3:1-2' status: green
    physical drive '3:1-3' status: green
    physical drive '3:1-4' status: green
    physical drive '3:1-5' status: green
    physical drive '3:1-6' status: green
checking host 'svt11.acme.com'
    status: alive
    logical drive '1' status: green
    logical drive '3' status: green
    physical drive '1:1-1' status: green
    physical drive '1:1-2' status: green
    physical drive '1:1-3' status: green
    physical drive '1:1-4' status: green
    physical drive '1:1-5' status: green
    physical drive '1:1-6' status: green
    physical drive '3:1-1' status: green
    physical drive '3:1-2' status: green
    physical drive '3:1-3' status: green
    physical drive '3:1-4' status: green
    physical drive '3:1-5' status: green
    physical drive '3:1-6' status: green
checking host 'svt12.acme.com'
    status: alive
    logical drive '1' status: green
    logical drive '3' status: green
    physical drive '1:1-1' status: green
    physical drive '1:1-2' status: green
    physical drive '1:1-3' status: green
    physical drive '1:1-4' status: green
    physical drive '1:1-5' status: green
    physical drive '1:1-6' status: green
    physical drive '3:1-1' status: green
    physical drive '3:1-2' status: green
    physical drive '3:1-3' status: green
    physical drive '3:1-4' status: green
    physical drive '3:1-5' status: green
    physical drive '3:1-6' status: green
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**